frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -421,6 +421,16 @@ export const OperatorHubTileView: React.FC<OperatorHubTileViewProps> = (props) =
     detailsItem &&
     `/k8s/ns/${detailsItem.subscription.metadata.namespace}/${SubscriptionModel.plural}/${detailsItem.subscription.metadata.name}?showDelete=true`;
 
+  let remoteWorkflowUrl = detailsItem && detailsItem.marketplaceRemoteWorkflow;
+  try {
+    const url = new URL(remoteWorkflowUrl);
```

Comment:
It looks like this tries to parse an undefined URL and spams the console with errors like

```
URL constructor: undefined is not a valid URL.
```

We should only try to parse the URL if defined. (Sorry I missed this on previous review.)

Reply:
No worries, I should have caught it. Thank you for the review!